ci/engine.go

```diff
@@ -41,14 +44,24 @@ func (e *Engine) WithGPUSupport() *Engine {
 	return e
 }
 
+func (e *Engine) WithDebug() *Engine {
+	e.Debug = true
+	return e
+}
```

Comment:
I think this makes `debug = false` the default? So our tests are now effectively running without debug logging? I remember we discussed this in #6755, but not sure if this is intentional here?

Reply:
Ah, yep looks like I kept it for `./hack/dev` but not the tests, will adjust!

Reply:
Actually this brings up another topic: our published engine _also_ as `debug = true` by default. Should we keep doing that?